docs/.sphinx/.markdownlint.json

```diff
@@ -0,0 +1,16 @@
+{
+    "default": false,
+    "MD003": { "style": "atx" },
+    "MD013": { "code_blocks": false, "tables": false, "stern": true, "line_length": 150},
```

Comment:
We have a variety of line lengths, I think for code we stick to 120. Do you think it would be worth it to be consistent?

Reply:
Good catch, thanks. I agree that we should be consistent. I will update the `line_length` here to 120